openstack/orchestration/v1/stacks/template_test.go

```diff
@@ -198,3 +198,92 @@ resources:
 	te.Parse()
 	th.AssertDeepEquals(t, expectedParsed, te.Parsed)
 }
+
+func TestGetFileContentsComposeRelativePath(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	baseurl, err := getBasePath()
+	th.AssertNoErr(t, err)
+
+	novaPath := strings.Join([]string{"templates", "my_nova.yaml"}, "/")
```

Comment:
TBH I was just following the basis of the other tests. But I think it should remain a forward slash regardless of the OS because it's actually meant to be a part of a file URL (internally, these will be converted fo `file:////...`). I don't think these use OS-dependent separators.